CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [1.2.1] - 2019-02-14
+
+### Fixed
+
+- Remove edX LTI consumer Xblock as a dependency as we cannot publish a package
```

Comment:
Or : 
as Pypi does not allow publishing a package with dependencies listed as PEP 508 URLs?